src/fundamental/tpm-pcr.c

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "macro-fundamental.h"
```

Comment:
What you need this for?

Reply:
it was an easy way to get the definition of NULL. will change to stddef.h